search/app.js

```diff
@@ -232,7 +247,9 @@ exports.lambda_handler = async (event, context) => {
   try {
     const path = event.path.replace(/\/$/, ''); // Remove last slash
     const { httpMethod } = event.requestContext;
-    const potentialMatch = _.chain(pathToFunction)
+    // TODO: remove lodash
+    // const potentialMatch = _.chain(pathToFunction)
+    const potentialMatch = pathToFunction
```

Comment:
This actually looks like it will work, but needs to be tested. Unfortunately, this code is not secluded enough to do that, so a manual test will have to be run.

I think the the best way to check is if a call to `/conformance` returns the correct response. The fail test should be gibberish will get a 404.